### ISSUE TITLE
[feature] Encrypt sensitive files on dotdrop import and update

### ISSUE THREAD

**Issue by clue44** [deadc0de6/dotdrop#190]
I was happy to see support for encrypting sensitive files in dotdrop. And on install the file can be decrypted which is very nice.

However, if I update the content of a sensitive file, its very easy to forget to do the manual steps needed for encrypting it. As a result, the unencrypted file ends up on Gitlab by mistake.

How would you feel about adding an option that will work as a "trans_write" (in opposite way of "trans_read")?

To give you an example, here is my config for decrypting an AWS credentials file:

```
dotfiles:
  f_credentials:
    src: aws/credentials
    dst: ~/.aws/credentials
    trans_read: gpg
trans_read:
  gpg: gpg2 -q --for-your-eyes-only --no-tty -d {0} > {1}
```

What if we could add something like trans_write to the config file:

```
dotfiles:
  f_credentials:
    src: aws/credentials
    dst: ~/.aws/credentials
    trans_read: gpg 
    trans_write: gpg

trans_read:
  gpg: gpg2 -q --for-your-eyes-only --no-tty -d {0} > {1}
trans_write:
  gpg: gpg2 -r a@b.com --armor --encrypt {1} > {0}
```

Basically run some command (like encrypt in this case) on a file when its being imported or updated. This would allow the user to not have to remember which files are sensitive and would avoid the mistake of pushing unencrypted files to gitlab. It could of course be used for a lot of other things. This is just one usecase.

**Comment by deadc0de6** [comment]
It does exist already, see [the dedicated wiki page](https://github.com/deadc0de6/dotdrop/wiki/usage-transformations) and [the config format page](https://github.com/deadc0de6/dotdrop/wiki/config#format).

See also this issue: #71 

If you have any trouble using it, let me know.
